java/ql/src/experimental/Security/CWE/CWE-078/CommandInjectionRuntimeExecLocal.ql

```diff
@@ -0,0 +1,24 @@
+/**
+ * @name Command Injection into Runtime.exec() with dangerous command
+ * @description High sensitvity and precision version of java/command-line-injection, designed to find more cases of command injection in rare cases that the default query does not find
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 6.1
+ * @precision high
+ * @id java/command-line-injection-extra-local
+ * @tags security
+ *       experimental
+ *       local
+ *       external/cwe/cwe-078
+ */
+
+import CommandInjectionRuntimeExec
+import ExecUserFlow::PathGraph
+
+from
+  ExecUserFlow::PathNode source, ExecUserFlow::PathNode sink, DataFlow::Node sourceCmd,
+  DataFlow::Node sinkCmd
+where callIsTaintedByUserInputAndDangerousCommand(source, sink, sourceCmd, sinkCmd)
+select sink, source, sink,
+  "Call to dangerous java.lang.Runtime.exec() with command '$@' with arg from untrusted input '$@'",
+  sourceCmd, sourceCmd.toString(), source.getNode(), source.toString()
```

Comment:
The two queries are identical.

Reply:
So something is wrong.

Reply:
Oh 🤦 good catch. The sources were moved to the `.qll` file so now both apply. So there _was_ a reason to have them in the `.ql` files @aegilops 😄

Reply:
That's why something I moved out wasn't in the `.qll` 🤦‍♂️

Tony asked, and I just thought, "yeah, I should put them in the QLL", without considering why I'd put it in there in the first place.

I'll move them back into the `.ql` to specialise them - great catch, thank you so much for the extra eyes on this!

Reply:
Done [here](https://github.com/github/codeql/pull/13484/commits/586c8803c57cbc82c7cf34361894ed73091ad42f), no worries :)